PURL-SPECIFICATION.rst

```diff
@@ -247,8 +247,9 @@ Use these rules for percent-encoding and decoding ``purl`` components:
 - the '#', '?', '@' and ':' characters must NOT be encoded when used as
   separators. They may need to be encoded elsewhere
```

Comment:
To fully address the confusion of #39, I would probably also change the last sentence here:

```suggestion
  separators. Some of them need to be encoded elsewhere as specified in the rules below.
```